arangod/RocksDBEngine/RocksDBMetadata.cpp

```diff
@@ -669,6 +669,43 @@ void RocksDBMetadata::loadInitialNumberDocuments() {
   return DocCount(0, 0, 0, RevisionId::none());
 }
 
+Result RocksDBMetadata::deleteCollectionMetaBatch(rocksdb::DB*, uint64_t objectId, rocksdb::WriteBatch &wb) {
```

Comment:
It seems the first function call argument (`DB*`) is not used. Shall we remove it from the signature?

```suggestion
Result RocksDBMetadata::deleteCollectionMetaBatch(uint64_t objectId, rocksdb::WriteBatch& wb) {
```